test/unit/store/cart-store.js

```diff
@@ -0,0 +1,31 @@
+import { Dispatcher } from 'consus-flux';
+import CartStore from '../../../.dist/store/cart-store';
+import { assert } from 'chai';
+
+describe('CartStore', () => {
+
+    it('should instantiate without any items', () => {
+        assert.strictEqual(CartStore.getItems().length, 0);
+    });
+
+    it('should add item to list', () => {
+        Dispatcher.handleAction('CHECKOUT_ITEM_FOUND',{
+            id: '123',
+            status: 'AVAILABLE'
+        });
+        assert.strictEqual(CartStore.getItems()[0].id, '123');
+        assert.strictEqual(CartStore.getItems()[0].status, 'AVAILABLE');
+    });
+
+    it('should clear items on checkout',() => {
+        Dispatcher.handleAction('CHECKOUT_ITEM_FOUND',{
+            id: '123',
+            status: 'AVAILABLE'
+        });
+
```

Comment:
Can you add an assert here to validate that an item is added? I know there is already a test for it but this test would still pass if the item is not added since it's length would be 0 already.